scripts/prepare_release.sh

```diff
@@ -72,10 +72,17 @@ function update_changelog() {
     fi
 }
 
+# create the release branch
+git fetch origin/master
+git rebase origin/master
```

Comment:
won't this rebase on the current branch, without checking it? is that the intended behavior of this script? It seems like the intent here is to work on the user's local master branch .

Reply:
yuuuup... missed an ever important checkout step 🤦